restore_dti.py example does not work in python3

```
restore_dti.py
Traceback (most recent call last):
  File "../tools/make_examples.py", line 122, in <module>
    exec(open(script).read(), namespace)
  File "<string>", line 39, in <module>
NameError: name 'reload' is not defined
make: *** [Makefile:122: rstexamples-stamp] Error 1
make html  6893.82s user 29.25s system 244% cpu 47:13.14 total
```

--- a/doc/examples/restore_dti.py
+++ b/doc/examples/restore_dti.py
@@ -27,6 +27,8 @@ dataset, which we will corrupt by adding intermittent noise.
 We start by importing a few of the libraries we will use.
 """
 
+from imp import reload
+
 import numpy as np
 import nibabel as nib
 

--- a/doc/examples/restore_dti.py
+++ b/doc/examples/restore_dti.py
@@ -27,8 +27,6 @@ dataset, which we will corrupt by adding intermittent noise.
 We start by importing a few of the libraries we will use.
 """
 
-from imp import reload
-
 import numpy as np
 import nibabel as nib
 
@@ -38,7 +36,6 @@ including an implementation of the RESTORE algorithm.
 """
 
 import dipy.reconst.dti as dti
-reload(dti)
 
 """
 ``dipy.data`` is used for small datasets that we use in tests and examples.

